recipes/mocknetworkaccessmanager/all/conanfile.py

```diff
@@ -0,0 +1,66 @@
+from conan import ConanFile
+from conan.tools.build import check_min_cppstd
+from conan.tools.cmake import CMake, CMakeDeps, CMakeToolchain, cmake_layout
+from conan.tools.files import apply_conandata_patches, copy, export_conandata_patches, get
+import os
+
+
+required_conan_version = ">=1.52.0"
+
+
+class MockNetworkAccessManagerConan(ConanFile):
+    name = "mocknetworkaccessmanager"
+    description = "Mocking network communication for Qt applications"
+    license = "MIT"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://gitlab.com/julrich/MockNetworkAccessManager"
+    topics = ("qt", "mock", "network", "QNetworkAccessManager", "unit test", "test")
+    package_type = "static-library"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "fPIC": [True, False],
```

Comment:
```suggestion
        "fPIC": [True, False],
        "with_qt": [5, 6],
```
Let's add an option to chose Qt version